azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -161,9 +166,9 @@ export class AzureClient {
         services: AzureContainerServices;
     }> {
         const loader = this.createLoader(containerSchema);
-        const url = new URL(this.props.connection.orderer);
-        url.searchParams.append("storage", encodeURIComponent(this.props.connection.storage));
-        url.searchParams.append("tenantId", encodeURIComponent(this.props.connection.tenantId));
+        const url = new URL(this.props.connection.endpoint);
+        url.searchParams.append("storage", encodeURIComponent(this.props.connection.endpoint));
```

Comment:
`endpoint` is getting appended as "storage" ?

Reply:
I honestly don't know either, all I know is that we supply the driver with endpoint URL in place of storage URL and it gets auto magically replaced 🤷